zerver/openapi/zulip.yaml

```diff
@@ -3168,23 +3184,14 @@ paths:
                                         - type: integer
                                         - type: string
                                   description: |
-                                    An array of tuples, where each tuple describes a linkifier.
-                                    The first element of the tuple is a
-                                    string regex pattern which represents the pattern that should
-                                    be linkified on matching. The second element is the URL with which the
-                                    pattern matching string should be linkified with and the third element
-                                    is the ID of the realm filter.
+                                    An array that is always empty. It is only intended
+                                    for linkifiers created using the legacy format, which no longer exists
+                                    after being replaced by linkifiers with
+                                    [RFC 6570](https://www.rfc-editor.org/rfc/rfc6570.html) compliant URL template.
```

Comment:
Then once the description of the overall event makes clear that the current state is that this is never sent at all, I think this description can be left as it was.

The information in the existing description should be kept around in any case — it's needed for clients supporting older servers. The alternative to keeping it in the main text of the description would be to move it into a "Changes" item.